diff -urNp strongswan-5.2.0dr4-patched/src/libstrongswan/plugins/plugin_loader.c strongswan-5.2.0dr4-current/src/libstrongswan/plugins/plugin_loader.c
--- strongswan-5.2.0dr4-patched/src/libstrongswan/plugins/plugin_loader.c	2014-05-22 16:14:28.740324392 -0400
+++ strongswan-5.2.0dr4-current/src/libstrongswan/plugins/plugin_loader.c	2014-05-22 16:15:20.172305012 -0400
@@ -354,7 +354,7 @@ static plugin_entry_t *load_plugin(priva
 			return NULL;
 		}
 	}
-	handle = dlopen(file, RTLD_LAZY);
+	handle = dlopen(file, RTLD_NOW|RTLD_GLOBAL);
 	if (handle == NULL)
 	{
 		DBG1(DBG_LIB, "plugin '%s' failed to load: %s", name, dlerror());
