From c282e8fa3c55a9d0046a3119d7b2a3fe07d83c37 Mon Sep 17 00:00:00 2001
From: Jamie Nguyen <j@jamielinux.com>
Date: Mon, 15 Jul 2013 22:31:34 +0100
Subject: [PATCH] Change 'ipsec scepclient' to 'strongswan scepclent'

---
 src/starter/starter.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/starter/starter.c b/src/starter/starter.c
index 917e52d..868b224 100644
--- a/src/starter/starter.c
+++ b/src/starter/starter.c
@@ -293,7 +293,7 @@ static void generate_selfcert()
 #endif
 		setegid(gid);
 		seteuid(uid);
-		ignore_result(system("ipsec scepclient --out pkcs1 --out cert-self --quiet"));
+		ignore_result(system("strongswan scepclient --out pkcs1 --out cert-self --quiet"));
 		seteuid(0);
 		setegid(0);
 
-- 
1.8.3.1

