Index: strongswan-4.6.0/init/systemd/strongswan.service.in
===================================================================
--- strongswan-4.6.0.orig/init/systemd/strongswan.service.in
+++ strongswan-4.6.0/init/systemd/strongswan.service.in
@@ -3,7 +3,7 @@ Description=strongSwan IPsec
 After=syslog.target
 
 [Service]
-ExecStart=@SBINDIR@/ipsec start --nofork
+ExecStart=@SBINDIR@/strongswan start --nofork
 StandardOutput=syslog
 
 [Install]
