diff --git a/src/checksum/Makefile.am b/src/checksum/Makefile.am
index d172b15..5dfab01 100644
--- a/src/checksum/Makefile.am
+++ b/src/checksum/Makefile.am
@@ -102,7 +102,7 @@ endif
 if USE_TOOLS
   exes += $(DESTDIR)$(ipsecdir)/openac
   exes += $(DESTDIR)$(ipsecdir)/scepclient
-  exes += $(DESTDIR)$(bindir)/pki
+  exes += $(DESTDIR)$(ipsecdir)/pki
 endif
 
 if USE_ATTR_SQL
diff --git a/src/ipsec/_ipsec.in b/src/ipsec/_ipsec.in
index 3c1f998..f39eb16 100644
--- a/src/ipsec/_ipsec.in
+++ b/src/ipsec/_ipsec.in
@@ -15,7 +15,7 @@
 # for more details.
 
 # define a minimum PATH environment in case it is not set
-PATH="/sbin:/bin:/usr/sbin:/usr/bin:@IPSEC_SBINDIR@:@IPSEC_BINDIR@"
+PATH="@IPSEC_DIR@:/sbin:/bin:/usr/sbin:/usr/bin:@IPSEC_SBINDIR@:@IPSEC_BINDIR@"
 export PATH
 
 # set daemon name
@@ -28,7 +28,6 @@ IPSEC_VERSION="U@IPSEC_VERSION@/K`uname -r`"
 
 # where the private directory and the config files are
 IPSEC_DIR="@IPSEC_DIR@"
-IPSEC_BINDIR="@IPSEC_BINDIR@"
 IPSEC_SBINDIR="@IPSEC_SBINDIR@"
 IPSEC_CONFDIR="@IPSEC_CONFDIR@"
 IPSEC_PIDDIR="@IPSEC_PIDDIR@"
@@ -40,7 +39,7 @@ IPSEC_CHARON_PID="${IPSEC_PIDDIR}/${DAEMON_NAME}.pid"
 IPSEC_STROKE="${IPSEC_DIR}/stroke"
 IPSEC_STARTER="${IPSEC_DIR}/starter"
 
-export IPSEC_DIR IPSEC_BINDIR IPSEC_SBINDIR IPSEC_CONFDIR IPSEC_PIDDIR IPSEC_SCRIPT IPSEC_VERSION IPSEC_NAME IPSEC_STARTER_PID IPSEC_CHARON_PID
+export IPSEC_DIR IPSEC_SBINDIR IPSEC_CONFDIR IPSEC_PIDDIR IPSEC_SCRIPT IPSEC_VERSION IPSEC_NAME IPSEC_STARTER_PID IPSEC_CHARON_PID
 
 IPSEC_DISTRO="Institute for Internet Technologies and Applications\nUniversity of Applied Sciences Rapperswil, Switzerland"
 
@@ -316,7 +315,7 @@ update)
 	;;
 pki)
 	shift
-	exec $IPSEC_BINDIR/pki "$@"
+	exec pki "$@"
 	;;
 version|--version)
 	printf "$OS_NAME $IPSEC_NAME $IPSEC_VERSION\n"
diff --git a/src/pki/Makefile.am b/src/pki/Makefile.am
index efbed9b..7835152 100644
--- a/src/pki/Makefile.am
+++ b/src/pki/Makefile.am
@@ -1,6 +1,6 @@
 SUBDIRS = man
 
-bin_PROGRAMS = pki
+ipsec_PROGRAMS = pki
 
 pki_SOURCES = pki.c pki.h command.c command.h \
 	commands/gen.c \
